crates/flox/doc/flox-show.md

```diff
@@ -0,0 +1,40 @@
+---
+title: FLOX-SHOW
+section: 1
+header: "flox User Manuals"
+...
+
+
+# NAME
+
+flox-show - show detailed information about a single package
+
+# SYNOPSIS
+
+flox [ `<general-options>` ] show `<name>` [ \--all ]
+
+# DESCRIPTION
+
+Show detailed information about a single package.
+
+The provided name must be an exact match for a package name i.e. it
+must be something you would have copied from the output of the
+`flox search` command.
+
+The default output includes the package description, name, latest version,
+and license. The `--all` flag will show all versions of the package
+that were found in the inputs listed in the manifest.
```

Comment:
Its a manpage from the future duh

Reply:
I wrote it this way so that we don't have to rewrite the manpage once we're actually using manifests. The user will list inputs in the manifest, correct?

Reply:
I'm not sure if we'll only use the active environment or merge that with global channels? For now we only have one input though, so `--all` doesn't even really make sense, right? Either way I think "the environment in the current directory" would be more clear than "the manifest" if that's what you mean

Reply:
`--all` will show multiple package versions when there are multiple versions available (I think there are 3-4 versions of `hello` in `nixpkgs`). It's also part of the GA spec.

`Current directory` won't make sense for managed environments, correct?

Reply:
I guess we will do a sweeping change over manpages anyway wrt noting the behavior related to the behavior dependent on "the current active environment, or otherwise the closest reachable environment".

Reply:
Ah wasn't thinking of that case in a single input.

I was thinking for a managed environment that the info in the current directory still determines the ultimate manifest

Reply:
Alright, let's just agree to revisit all of the manpages before GA so that they reflect reality at that time.